apps-rendering/src/components/editions/byline/index.tsx

```diff
@@ -119,27 +120,33 @@ const styles = (
 };
 
 const largeTextStyles = (
-	fontStyle: FontStyle,
-	fontWeight: FontWeight,
-	lineHeight?: LineHeight,
+	fontStyle: 'normal' | 'italic',
+	fontWeight: 'light' | 'bold',
 ): SerializedStyles => css`
-	${headline.xsmall({ fontStyle, fontWeight, lineHeight })};
-
+	${fontStyle === 'italic' && fontWeight === 'light'
+		? headlineLightItalic24
+		: headlineBold24};
+	line-height: 1.3;
 	${from.tablet} {
-		${headline.small({ fontStyle, fontWeight, lineHeight })};
+		${fontStyle === 'italic' && fontWeight === 'light'
+			? headlineLightItalic28
+			: headlineBold28};
+		line-height: 1.3;
 	}
-
 	${from.desktop} {
-		${headline.medium({ fontStyle, fontWeight, lineHeight })};
+		${fontStyle === 'italic' && fontWeight === 'light'
+			? headlineLightItalic34
+			: headlineBold34};
+		line-height: 1.3;
 	}
 `;
 
 const standardTextStyles = (
-	fontStyle: FontStyle,
-	fontWeight: FontWeight,
-	lineHeight: LineHeight,
+	fontStyle: 'normal' | 'italic',
+	fontWeight: 'light' | 'bold',
 ): SerializedStyles => css`
-	${body.medium({ fontStyle, fontWeight, lineHeight })}
+	${fontStyle === 'italic' ? textEgyptianBoldItalic17 : textEgyptianBold17};
```

Comment:
It really needs `textEgyptianLightItalic17` which doesn't exist, hence the `font-weight` override below. I've updated this to match `largeTextStyles` above with a single boolean argument and changed the preset to `textEgyptianItalic17` which is the closest we have. (Removing the override and leaving `font-weight` as 400 instead of 300 probably wouldn't be that noticeable, but I've kept it for now so there are no visual differences.)